src/editor/clipboard.js

```diff
@@ -33,6 +34,9 @@ export default class Clipboard {
     item.getAsString((text) => {
       if (isUrl(text) && this.contents.hasSelectedText()) {
         this.contents.createLinkWithSelectedText(text)
+      } else if (isUrl(text)) {
+        const nodeKey = this.contents.createLink(text)
+        dispatchLinkEvent("created", this.editorElement, nodeKey, { url: text })
```

Comment:
Minor: I think this should be a regular private method in the clipboard, or maybe make the helper a generic `events_helper.js`. These helper tend to be generic in nature and cross-cutting.